src/explore-education-statistics-admin/src/pages/admin-dashboard/components/__tests__/ScheduledReleasesTable.test.tsx

```diff
@@ -145,7 +145,8 @@ describe('ScheduledReleasesTable', () => {
             {
               ...testReleases[0],
               permissions: {
-                ...testReleases[0].permissions,
+                // eslint-disable-next-line @typescript-eslint/no-non-null-assertion
+                ...testReleases[0].permissions!,
```

Comment:
If we separate the permissions out into a separate variable with an explicit type, we don't need this non-null assertion